lighthouse-core/report/html/templates.html

```diff
@@ -500,6 +500,9 @@
 
     .lh-gauge__svg-wrapper {
       position: relative;
+      /* The wrapper adds some height to the element. display: inline-block doesn't
+          work for some reason, so instead just shave off the extra height. */
+      margin-bottom: -7px;
```

Comment:
hmmm i'm only seeing 5 extra pixels, not 7. 

also i'd much rather determine where the phantom height is coming from and address that rather than go with negative margin hacks.

Reply:
![image](https://user-images.githubusercontent.com/4071474/56758612-f90b8280-674b-11e9-83b6-79e10f4aea2e.png)
![image](https://user-images.githubusercontent.com/4071474/56758632-0163bd80-674c-11e9-99bd-93ac70ca8a1c.png)

I tried setting the svg wrapper to inline-block (which should not consume those extra 7 pixels):

![image](https://user-images.githubusercontent.com/4071474/56758679-150f2400-674c-11e9-8bba-72d4344dc142.png)

But it gets ignored.

...After typing this, I realized I needed to set the height: `height: var(--gauge-circle-size)`.